xapian-core/api/omenquire.cc

```diff
@@ -452,6 +464,23 @@ MSet::Internal::read_docs() const
     requested_docs.clear();
 }
 
+void
+MSet::Internal::set_new_weights(double wt, int i) {
```

Comment:
I am not able to set the weight of the particular item as `item[i].wt = wt`that is why I am using item_set_weight()